### ISSUE TITLE
[Package Issue]: Oracle.JavaRuntimeEnvironment - No applicable update found when one does exist

### ISSUE THREAD

**Issue by EmeraldFlame** [microsoft/winget-pkgs#49395]
### Please confirm these before moving forward

- [X] I have searched for my issue and not found a work-in-progress/duplicate/resolved issue.
- [X] I have not been informed if the issue is resolved in a preview version of the winget client.

### Category of the issue

No applicable update found.

### Brief description of your issue

Winget shows a new version of JRE when using list command, but when attempting to upgrade to it, returns with 'No applicable update found.'

### Steps to reproduce

1. Open Windows Terminal as administrator
2. winget list Oracle.JavaRuntimeEnvironment
3. winget upgrade Oracle.JavaRuntimeEnvironment

### Actual behavior

winget upgrade Oracle.JavaRuntimeEnvironment returns with "No applicable update found."

### Expected behavior

winget upgrade Oracle.JavaRuntimeEnvironment would upgrade JRE to the latest version

### Environment

```shell
Windows Package Manager v1.1.13405
Windows: Windows.Desktop v10.0.22000.527
Package: Microsoft.DesktopAppInstaller v1.16.13405.0
Current Install: Java 8 Oracle.JavaRuntimeEnvironment 8.0.3110.11
Available: Java 8 Oracle.JavaRuntimeEnvironment 8.0.3210.7
```

### Screenshots and Logs

[winget list Oracle.JavaRuntimeEnvironment.log](https://github.com/microsoft/winget-pkgs/files/8147474/WinGet-2022-02-26-15-26-12.267.log)
[winget upgrade Oracle.JavaRuntimeEnvironment.log](https://github.com/microsoft/winget-pkgs/files/8147475/WinGet-2022-02-26-15-26-19.919.log)

![winget upgrade Orace JavaRuntimeEnvironment error](https://user-images.githubusercontent.com/13631454/155858648-e1bf6e51-0b8b-4643-8379-08a7050ce56e.png)

**Comment by OfficialEsco** [comment]
Hey and thank you for using the issue template as intended! :)
The issue is resolved in the [Windows Package Manager 1.3.432-preview](https://github.com/microsoft/winget-cli/releases/tag/v1.3.431) client, the issue is caused by the Installer unpacking another Installer (in this case a MSI) which makes WinGet think you got the MSI version installed instead of a regular EXE (f.ex Nullsoft/Inno)

The current Manifest is specced out for this feature so we're just waiting for winget-cli to catch up :)
https://github.com/microsoft/winget-pkgs/blob/777c8230620f6bb5e40412cd348daeac426127ca/manifests/o/Oracle/JavaRuntimeEnvironment/8.0.3210.7/Oracle.JavaRuntimeEnvironment.installer.yaml#L29-L47

**Comment by EmeraldFlame** [comment]
Awesome, thanks for the quick reply. I had looked through the preview patch notes but must have missed that. Thanks for letting me know!
